packages/connectivity/connectivity/ios/Classes/FLTConnectivityPlugin.m

```diff
@@ -139,6 +106,46 @@ - (void)onReachabilityDidChange:(NSNotification*)notification {
   _eventSink([self statusFromReachability:curReach]);
 }
 
+- (NSString*)getWifiIP {
```

Comment:
I would put the code back in the old position. By moving it, there's no easily ability to see if the code has been edited. Leaving it in the old position saves a single `#ndef` which is a very modest cost.

Reply:
moved all blocks as close as possible to their old positions